src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -90,6 +88,16 @@ public ReturnCountCheck() {
     }
 
     /**
+     * Set the format to the specified regular expression.
```

Comment:
I think we should be a bit more descriptive here - what does `format` mean in context of ReturnCountCheck?